backend/src/postsDataSource.ts

```diff
@@ -57,24 +56,23 @@ export class PostsDataSource extends DataSource {
     const foundUser = this.getUsers().find(user => user.email === email)
 
     if (foundUser) {
-      if (bcrypt.compareSync(password, foundUser.password)) {
-        return jwt.sign({ id: foundUser.id }, process.env.JWTSECRET, { algorithm: 'HS256' })
+      if (compareSync(password, foundUser.password)) {
+        return sign({ id: foundUser.id }, JWTSECRET, { algorithm: 'HS256' })
       }
+      throw new AuthenticationError('Wrong email/password combination')
```

Comment:
I think this should be below the outmost `if` statement. What if `foundUser` is falsy?